code/modules/mob/living/simple_animal/abnormality/teth/pale_horse.dm

```diff
@@ -0,0 +1,221 @@
+						ABNORMALITY_WORK_INSTINCT = 50,
+						ABNORMALITY_WORK_INSIGHT = 30,
+						ABNORMALITY_WORK_ATTACHMENT = 45,
+						ABNORMALITY_WORK_REPRESSION = 50
+						)
+	work_damage_amount = 5
+	work_damage_type = PALE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/revelation,
+		/datum/ego_datum/armor/revelation
+		)
+	gift_type =  /datum/ego_gifts/revelation
+	gift_message = "Your eyes have been opened."
+
+	//Quiploth counter
+	var/death_counter = 0
+	//teleport
+	var/can_act = TRUE
+	var/teleport_cooldown
+	var/teleport_cooldown_time = 30 SECONDS
+	//attack
+	var/pulse_range = 5 //from 5 up to 45 pale damage if caught by 9 clouds at once
+
+//work stuff
+/mob/living/simple_animal/hostile/abnormality/pale_horse/SuccessEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/Worktick(mob/living/carbon/human/user)
+	if(user.health < (user.maxHealth * 0.5))
+		return
+	else
+		user.apply_damage(4, PALE_DAMAGE, null, user.run_armor_check(null, PALE_DAMAGE))
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/proc/on_mob_death(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(!(status_flags & GODMODE)) // If it's breaching right now
+		return FALSE
+	if(!ishuman(died))
+		return FALSE
+	if(!died.ckey)
+		return FALSE
+	death_counter += 1
+	if(death_counter >= 2)
+		death_counter = 0
+		datum_reference.qliphoth_change(-1)
+	return TRUE
+
+//Breach stuff
+/mob/living/simple_animal/hostile/abnormality/pale_horse/proc/TryTeleport() //stolen from knight of despair
+	dir = 2
+	if(teleport_cooldown > world.time)
+		return FALSE
+	teleport_cooldown = world.time + teleport_cooldown_time
+	var/list/teleport_potential = list()
+	for(var/turf/T in GLOB.xeno_spawn)
+		teleport_potential += T
+	if(!LAZYLEN(teleport_potential))
+		return FALSE
+	var/turf/teleport_target = pick(teleport_potential)
+	animate(src, alpha = 0, time = 5)
+	new /obj/effect/temp_visual/guardian/phase(get_turf(src))
+	SLEEP_CHECK_DEATH(5)
+	animate(src, alpha = 255, time = 5)
+	new /obj/effect/temp_visual/guardian/phase/out(teleport_target)
+	forceMove(teleport_target)
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/Life()
+	. = ..()
+	if(status_flags & GODMODE)
+		return
+	SpawnFog() //Periodically spews out damaging fog while breaching
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/proc/SpawnFog()
+	var/turf/target_turf = get_turf(src)
+	for(var/turf/T in view(1, target_turf))
+		if(prob(70))
+			continue
+		new /obj/effect/palefog(T)
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/Move() //more damaging fog when moving
+	var/turf/target_turf = get_turf(src)
+	for(var/turf/T in view(0, target_turf))
+		new /obj/effect/palefog(T)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/AttackingTarget()
+	if(!isliving(target))
+		return FALSE
+
+	var/mob/living/L = target
+	if((L.health > 0 || L.stat == DEAD))
```

Comment:
So it ignores dead people and dusts crit people?

Reply:
This might be a typo, i'll look at it and fix.

Reply:
Still broke, need to spend more time on the AI and pathing.